DependencyInjection/CmfRoutingExtension.php

```diff
@@ -282,6 +282,12 @@ private function loadOrmProvider($config, XmlFileLoader $loader, ContainerBuilde
 
         $container->setParameter('cmf_routing.backend_type_orm', true);
         $container->setParameter('cmf_routing.dynamic.persistence.orm.manager_name', $config['manager_name']);
+        $container->setParameter('cmf_routing.dynamic.persistence.orm.route_class', $config['route_class']);
+        if (false !== strpos($config['route_class'], 'Symfony\Cmf\Bundle\RoutingBundle\Doctrine\Orm')) {
```

Comment:
why not an exact string comparison for the ORM route class?

and if we can do that, please use the ::class notation instead of a string for the class name